optaplanner-examples/src/main/java/org/optaplanner/examples/meetingscheduling/persistence/MeetingSchedulingXlsxFileIO.java

```diff
@@ -0,0 +1,825 @@
+            nextCell().setCellValue(""); // TODO: add a meetingName field to class MeetingSchedule?
+            nextCell().setCellValue(DAY_FORMATTER.format(LocalDateTime.now()) + " " + TIME_FORMATTER.format(LocalDateTime.now()));
+            nextRow();
+            nextRow();
+            nextHeaderCell("Constraint");
+            nextHeaderCell("Weight");
+            nextHeaderCell("Description");
+
+            for (MeetingSchedulingConstraints constraint : MeetingSchedulingConstraints.values()) {
+                writeIntConstraintLine(constraint.getConstraintName(), constraint::getConstraintWeight, "");
+                if (constraint.equals(MeetingSchedulingConstraints.ENTIRE_GROUP_MEETING_NOT_SCHEDULED)
+                        || constraint.equals(MeetingSchedulingConstraints.PREFERRED_ATTENDANCE_CONFLICT)) {
+                    nextRow();
+                }
+            }
+
+            autoSizeColumnsWithHeader();
+        }
+
+        private void writePersons() {
+            nextSheet("Persons", 1, 0, false);
+            nextRow();
+            nextHeaderCell("Full name");
+            for (Person person : solution.getPersonList()) {
+                nextRow();
+                nextCell().setCellValue(person.getFullName());
+            }
+            autoSizeColumnsWithHeader();
+        }
+
+        private void writeMeetings() {
+            nextSheet("Meetings", 1, 1, false);
+            nextRow();
+            nextHeaderCell("Topic");
+            nextHeaderCell("Group");
+            nextHeaderCell("Duration");
+            nextHeaderCell("Speakers");
+            nextHeaderCell("Content");
+            nextHeaderCell("Required attendance list");
+            nextHeaderCell("Preferred attendance list");
+            for (Meeting meeting : solution.getMeetingList()) {
+                nextRow();
+                nextCell().setCellValue(meeting.getTopic());
+                nextCell().setCellValue(meeting.isEntireGroupMeeting() ? "Y" : "");
+                nextCell().setCellValue(meeting.getDurationInGrains() * TimeGrain.GRAIN_LENGTH_IN_MINUTES);
+                nextCell().setCellValue(meeting.getSpeakerList() == null ? "" :
+                                                meeting.getSpeakerList().stream()
+                                                        .map(Person::getFullName)
+                                                        .collect(joining(", ")));
+                nextCell().setCellValue(meeting.getContent() == null ? "" : meeting.getContent());
+                nextCell().setCellValue(
+                        String.join(", ", meeting.getRequiredAttendanceList().stream()
+                                .map(requiredAttendance -> requiredAttendance.getPerson().getFullName())
+                                .collect(toList())));
+                nextCell().setCellValue(
+                        String.join(", ", meeting.getPreferredAttendanceList().stream()
```

Comment:
Nit-picking, but these two joins can be replaced by `joining(", ")` collector, which is already used on different places in this class and is nicer IMO.